lib/live_debugger/components/error.ex

```diff
@@ -14,6 +16,7 @@ defmodule LiveDebugger.Components.Error do
         <%= render_slot(@heading) %>
       </div>
       <p class="mb-4"><%= render_slot(@description) %></p>
+      <.report_issue />
```

Comment:
Report issue should only be in `unexpected_error_component` the rest of them should not be reported